docs/evaluate/temporal-cloud/gcp-private-service-connect.mdx

```diff
@@ -58,10 +58,12 @@ Set up Private Service Connect with Temporal Cloud with these steps:
 
   - For **Target service**, fill in the **Service name** with the Private Service Connect Service Name for the region you’re trying to connect to:
 
-      | Region                 | Private Service Connect Service Name                                                      |
-      | ---------------------- | ----------------------------------------------------------------------------------------- |
-      | `asia-south1`          | `projects/prod-d5spc2sfeshws33bg33vwdef7/regions/asia-south1/serviceAttachments/pl-7w7tw` |
-      | `us-west1   `          | `projects/prod-rbe76zxxzydz4cbdz2xt5b59q/regions/us-west1/serviceAttachments/pl-94w0x`    |
+      | Region         | Private Service Connect Service Name                                                       |
+      | -------------- | ------------------------------------------------------------------------------------------ |
+      | `asia-south1`  | `projects/prod-d5spc2sfeshws33bg33vwdef7/regions/asia-south1/serviceAttachments/pl-7w7tw`  |
+      | `europe-west3` | `projects/prod-kwy7d4faxp6qgrgd9x94du36g/regions/europe-west3/serviceAttachments/pl-acgsh` |
+      | `us-east4`     | `projects/prod-y399cvr9c2b43es2w3q3e4gvw/regions/us-east4/serviceAttachments/pl-8awsy`     |
+      | `us-west1   `  | `projects/prod-rbe76zxxzydz4cbdz2xt5b59q/regions/us-west1/serviceAttachments/pl-94w0x`     |
```

Comment:
Asking just based on superficially seeing regions and "private service connect" in both places, but should https://github.com/temporalio/documentation/blob/main/docs/production-deployment/cloud/references/regions/private-service.md?plain=1 also contain the regions listed here?